cargo-ros-install/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Licensed under the Apache License, Version 2.0
```

Comment:
Since this is a pure Rust project (there's no `ament` or `ROS` involved), can this be build with `colcon-cargo`? If so, CMake wouldn't be necessary here.

Reply:
It has been split off, and I deleted CMakeLists.txt and package.xml. I unthinkingly just pushed to master of the new repo instead of opening a PR. Let me know if I should add a PR in order to review it, or whether you're fine just reading through the repo.